tests/add-delete-add.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from '@playwright/test';
+
+test('testing adding interventions, then deleting one, then adding another', async ({ page }) => {
+    await page.goto('/scheme.html?authority=Derby#16.84/52.906457/-1.504519');
+    await page.getByRole('region', { name: 'Map' }).waitFor();
```

Comment:
Do an explicit `.waitFor()` here to make sure the map loads else I found it got itself in a pickle and timed out.